### ISSUE TITLE
[COGNITION, SEMANTIC]: Radio inputs are to be used for groups of options where only one choice may be selected

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#4526]
# [COGNITION, SEMANTIC]: Radio inputs are to be used for groups of options where only one choice may be selected

## Description

[SEMANTIC] In this mockup there is data presented with a single radio input. Radio inputs should be presented in a group of options where only one may be selected.
[COGNITION] In this area's design, there is a radio input along with a text link within the text to edit the address, a button to Update, and another button to Cancel. This presentation is confusing. I am not sure what the radio input is for, what the Edit Address link will do, what Update is for. The message above requests that I add a unit number. The error message asks that I add a unit number, so I need to update/edit the address.

<img width="412" alt="authd-exp-address-validation-ui-question01" src="https://user-images.githubusercontent.com/57469/71729156-0285d800-2e0d-11ea-9cb3-57fb2d7a571f.png">

## Point of Contact

Jennifer

## Acceptance Criteria

As a user, I want to readily understand the actions available to me so that I may confidently solve an issue with my address.

## Possible Fixes (optional)

Without context for why this was designed like this, I'm guessing at this fix. It appears that the radio input and Edit Address link could be removed, and only the buttons for Update and Cancel provided. Update would allow the user to edit the address to include a unit number.

## WCAG or Vendor Guidance (optional)

* [Radio inputs should be in groups of options](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)

**Comment by Samara-Strauss** [comment]
This design was intentional. People have two options here — to save the address they originally entered, which might be problematic (but might not be — the user knows best) or to edit the address and update it. Without the radio button, there's no way to really acknowledge that you've selected this address and intend to save it as-is. We talked about having the radio button pre-filled in instead of forcing the click, but went with the forced click to force users to be just a little bit more intentional about things.

And we can't remove the Edit link because someone might want to go back and change what they entered.

Let me know if this is still a problem given the context. At most, I'm willing to have the radio button pre-filled in and not force the click, but it doesn't make sense to remove the radio button.

**Comment by jenstrickland** [comment]
This is still a cognition issue, as well as a semantic markup issue. It may be clearer to present the options explicitly. 

- Use the address as entered
- Edit address

**Comment by Samara-Strauss** [comment]
We are not going to change the CTAs as they are consistent with the rest of the design/use cases.

**Comment by Samara-Strauss** [comment]
Actually, I would consider an update to the "update" button, but we are not going to change the Edit Address link.

**Comment by jenstrickland** [comment]
I'm sorry, I guess the bulleted list wasn't clear.

* radio input - Use the address as entered
* radio input - Edit the address

Button - Update (for input 1 — proceed; for input 2 — go back to edit)
Button - Cancel

**Comment by Samara-Strauss** [comment]
@jenstrickland what do you think of this? I think it addresses the radio button issue while also not breaking too far from the overall pattern we have set up for the rest of our use cases. I'm hoping it will both work for people who do and do not need screenreaders.

BTW button copy for the main CTA isn't final. If this design is good, I'll work with copy to get the final wording.

![13  Scenario 2_ Override; No suggestions](https://user-images.githubusercontent.com/34068740/71733638-1420ad00-2e18-11ea-8bc1-aa9012dfb636.png)

**Comment by jenstrickland** [comment]
This is much clearer. Thanks! 👍

**Comment by Samara-Strauss** [comment]
Great. I will work with copy to get button copy updated.

**Comment by Samara-Strauss** [comment]
Working on copy in #4529

**Comment by Samara-Strauss** [comment]
Closing. Opened FE ticket #4632 to resolve this.
